README.md

```diff
@@ -23,40 +29,22 @@
 
 
 To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+1. Clone the repository: https://github.com/JOdiemo/leaderboard/tree/leaderboard-final
+2. Copy to your local disc: you@your-Pc-name:~$ cd <folder>
```

Comment:
- In order to make your repo look more professional I'd suggest that you include steps on how to run this project locally 👍 👍 .

Reply:
well resolved! ✔️